File: MS.NET/Sarvam Backend/Repository/RepositoryImplementations/UserCredentialsRepository.cs
using Sarvam.Data;
using Sarvam.Models;
using Sarvam.Repository.RepositoryInterfaces;
using System.Threading.Tasks;

namespace Sarvam.Repository.RepositoryImplementations
{
    public class UserCredentialsRepository : IUserCredentialsRepository
    {
        private readonly ApplicationDbContext _context;

        public UserCredentialsRepository(ApplicationDbContext context)
        {
            _context = context;
        }

        public async Task AddUserAsync(UserCredentials user)
        {
            _context.Users.Add(user);
            await _context.SaveChangesAsync();
        }

        public async Task SaveChangesAsync()
        {
            await _context.SaveChangesAsync();
        }
    }
}

File: Sarvam C-DAC Project/Sarvam C-DAC Spring Boot/Sarvam/src/main/java/com/cdac/sarvam/repository/UserCredentialsRepository.java
package com.cdac.sarvam.repository;

import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

import com.cdac.sarvam.model.UserCredentials;


@Repository
public interface UserCredentialsRepository extends JpaRepository<UserCredentials, Long> {
}
